metadata-ingestion/src/datahub/ingestion/source/looker/looker_source.py

```diff
@@ -887,6 +887,7 @@ def _input_fields_from_dashboard_element(
                         schemaField=LookerUtil.view_field_to_schema_field(
                             view_field_for_reference,
                             self.reporter,
+                            True,
```

Comment:
I'm probably missing context here, but is the issue that the label field was only added to the SchemaField model recently? If so, why can this param be set to true?